### ISSUE TITLE
Disable scrolling

### ISSUE THREAD

**Issue by cbcto** [maaaaark/bcSwipe#5]
Thanks a lot for building this.
How can I disable accidental vertical scrolling, wen the user is trying to swipe the carousel left/right?

Thanks

**Comment by hommealone** [comment]
Up-vote. Again, thanks. Preventing vertical scrolling while swiping would be great. Does anyone know how to do that?

**Comment by avxkim** [comment]
+1, it has usability issues because of vertical scrolling.

**Comment by nuljon** [comment]
agree, please provide for no-scroll = true while swiping left or right
